v1/styles.css

```diff
@@ -407,6 +426,12 @@ ul {
   padding: 0;
 }
 
+.section-header__item-right {
+  text-align: right;
```

Comment:
a var should be set in `section-header__item` for lines 430 & 431. Update variable in `right` instead.

Reply:
I don't understand what you mean.